README.md

```diff
@@ -28,7 +28,7 @@ $ npm install -g @google/repo
 You need to make your own [`config.yaml`](https://github.com/googleapis/github-repo-automation/blob/master/config.yaml.default) and put your GitHub token there. You can set the path to the config file with the `REPO_CONFIG_PATH` environment variable:
 
 ```sh
-$ cat /User/beckwith/config.yaml
+$ cat > /User/beckwith/config.yaml
```

Comment:
I think the intention of this snippet was to show the contents of `config.yaml`, not to create a `config.yaml` perhaps it would be better to just put:

```md
**Example configuration:**


```yaml
githubToken: your-github-token
clonePath: /User/beckwith/.repo # optional
```
```

Reply:
done